### DISCUSSION TITLE
how to add number type support for value in select ?

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#5703] [Q&A]
![image](https://github.com/mantinedev/mantine/assets/83704092/55ebb8ee-cdaa-4dc6-baf4-469c5f06798c)

how to add number type support for value in select? 
Now if you pass to data=[{label:'test',value:1}] the component crashes

**Comment by rtivital** [comment]
Select does not support numbers or any other data types that are not strings in the data array. If you need this feature, you can either perform a conversion of types on your side or build a custom component with Combobox that handles this case.

**Reply by m1x-developer** [reply]
tnx
